src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,134 @@
+package duke;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Ui is a class that the user interacts with on the CLI
+ */
+public class Ui {
+
+    TaskList tasks;
+
+    Ui(TaskList tasks) {
+        this.tasks = tasks;
+    }
+
+    enum Commands {
+        LIST("list"),
+        MARK("mark"),
+        UNMARK("unmark"),
+        TODO("todo"),
+        DEADLINE("deadline"),
+        EVENT("event"),
+        DELETE("delete"),
+        FIND("find");
+
```

Comment:
The way you used enums was appropriate and relevant to the project. Additionally, you followed the coding standard and used PascalCase to name this enum rather than camelCase. 👍